client/src/Views/Reports.vue

```diff
@@ -82,6 +79,15 @@ export default {
     handleAllShoutsViewData: async function () {
       this.shouts = shoutoutFormatter(await getAllShouts());
     },
+    // handleRankedUserData: async function () {
+    //   const { selectedMonth, selectedYear } = this.createDateObj();
+    //   const formattedShouts = await getRankedByMonth(
+    //     "shouter",
+    //     selectedMonth,
+    //     selectedYear
+    //   );
+    //   this.shoutsToDisplay = rankedShoutersFormatter(formattedShouts);
+    // },
```

Comment:
Do you need this?

Reply:
In my PR, all this logic that is specific to a tab is broken out into it's proper component

Reply:
I mention it because it's commented out

Reply:
It is still used I believe